packages/distribution/addon/components/cd-navigation/controls.js

```diff
@@ -7,16 +6,55 @@ import { confirm } from "ember-uikit";
 import { gql } from "graphql-tag";
 
 import { decodeId } from "@projectcaluma/ember-core/helpers/decode-id";
+import completeWorkItemMutation from "@projectcaluma/ember-distribution/gql/mutations/complete-work-item.graphql";
+import inquiryIncompleteQuery from "@projectcaluma/ember-distribution/gql/queries/inquiry-incomplete.graphql";
```

Comment:
Let's name it `incomplete-inquiries.graphql`

Reply:
done